connectors/vault/vault_utils/vault_utils.go

```diff
@@ -43,36 +45,36 @@ func getEnvWithDefault(key string, defaultValue string) string {
 }
 
 type VaultConfig struct {
-	token   string
-	address string
+	Token   string
+	Address string
 }
 
 type VaultConnection struct {
-	config VaultConfig
-	client *api.Client
+	Config VaultConfig
+	Client *api.Client
 }
 
 func CreateVaultConnection() VaultConnection {
-	token := getEnv(VaultSecretKey)
-	address := getEnv(VaultAddressKey)
+	token := GetEnv(VaultSecretKey)
```

Comment:
Are you sure that this won't log the secret? As you do have a print command in the `GetEnv` method...

Reply:
fixed. Please check if the fix is ok. Thanks.